ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -201,25 +200,32 @@ const DeviceColumn = ({ deviceData, devices, idPrefix, handleRequestChange, lock
     );
 };
 
-const FormatColumn = ({ deviceData, handleRequestChange, idPrefix, request }) => {
+const FormatSwitch = ({ handleRequestChange, idPrefix, request, isRootMountPoint }) => {
+    return (
+        <Switch
+          id={idPrefix + "-switch"}
+          isChecked={request.reformat}
+          isDisabled={isRootMountPoint}
+          defaultChecked={isRootMountPoint}
```

Comment:
defaultChecked is not a property documented in the patternfly docs

Reply:
Here I encountered a problem compared to the tests. While the checkbox was in the default state false, the newly created switch is true, so I wanted to set the state of the checkbox so that the disks are not formatted by default.

Reply:
you just need to make sure to 'cast' the request.format to boolean.Do `isChecked={!!request.reformat}`

Reply:
I thought that request.reformat should provide this functionality, but it doesn't seem to work as well as I need for new mount locations